src/ui/src/app/charts-filters/charts-filters.component.ts

```diff
@@ -1,48 +0,0 @@
-import { Component, Input, OnInit, Output, EventEmitter } from '@angular/core';
```

Comment:
I see you added all the filter logic to `charts` component. Now, the `charts` component is bigger and I don't see any advantages to merge both files instead of having two separate components.